components/image-builder-api/go/config/config.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package config
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	//wsmanapi "github.com/gitpod-io/gitpod/ws-manager/api"
+	"golang.org/x/xerrors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"io/ioutil"
+)
+
+type Service struct {
+	Addr string    `json:"address"`
+	TLS  TLSConfig `json:"tls"`
```

Comment:
I don't think we support serving pprof over TLS

Reply:
That was done for simplicity, but I agree it makes it more complex - have created a new `type`